src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
If we release physics and entities emit a collide event, people might try to do `<a-collide>` when that doesn't exist. Better to accustom people to handle any event via `<a-event name="collide">`.